devicemodel/samples/nuc/launch_xenomai.sh

```diff
@@ -0,0 +1,219 @@
+#!/bin/bash
```

Comment:
Please add the copyright and license information (check the other launch script files for examples)